src/commands/command-fns/thanks.js

```diff
@@ -27,6 +28,23 @@ async function thanks(message) {
       `You have to use the word "for" when thanking someone. ${example}`,
     )
   }
+
+  thankedMembers.forEach(thankedMember => {
+    if (thanksHistory[thankedMember.id]) {
+      thanksHistory[thankedMember.id] += 1
+    } else {
+      thanksHistory[thankedMember.id] = 1
+    }
+  })
+
+  try {
+    await saveThanksHistory(thanksHistory)
+  } catch (_) {
```

Comment:
We should be able to do this:

```suggestion
  } catch {
```

Reply:
Did that not work?